src/bin/collapse-perf.rs

```diff
@@ -80,6 +80,9 @@ struct Opt {
     #[structopt(value_name = "PATH")]
     /// Perf script output file, or STDIN if not specified
     infile: Option<PathBuf>,
+
+    #[structopt(long = "skip-after", value_name = "STRING")]
```

Comment:
We should document what this flag does :)

Reply:
fixed